src/models/Tweet.ts

```diff
@@ -0,0 +1,48 @@
+import { ObjectId } from "mongodb";
+import { model, Schema, Document } from "mongoose";
+
+export interface Tweet extends Document {
+  //reference
+  authorId: ObjectId;
+  content: string;
+  createdAt: string;
+  repliedTo?: ObjectId;
+  updatedAt: string;
+  stats: {
+    likes: number;
+    retweets: Array<ObjectId>;
+    //Comments - todo, comments base
+  };
+}
+
+const TweetSchema: Schema = new Schema<Tweet>({
+  authorId: {
+    type: ObjectId,
+    required: true,
+  },
```

Comment:
```suggestion
  authorId: {
    type: Schema.Types.ObjectId,
    required: true,
    ref: "User",
  },
```
add ref to Schema and change ObjectId => Schema.Types.ObjectId,
you can see example in [Bookmark.model.ts](https://github.com/attech-org/leleka-backend/blob/develop/src/models/Bookmark.model.ts)

[use ref](https://mongoosejs.com/docs/populate.html)